src/content/app/genome-browser/services/genome-browser-service/genomeBrowserCommands.ts

```diff
@@ -59,26 +59,26 @@ export const setBrowserLocation = (payload: {
 
 export const toggleTrack = (payload: {
   genomeBrowser: EnsemblGenomeBrowser;
-  trackId: string;
+  trackPath: string[];
   isEnabled: boolean;
 }) => {
-  const { genomeBrowser, trackId, isEnabled } = payload;
-  genomeBrowser.switch(['track', trackId], isEnabled);
+  const { genomeBrowser, trackPath, isEnabled } = payload;
+  genomeBrowser.switch(trackPath, isEnabled);
 };
 
 // NOTE: this method can only handle boolean flags
```

Comment:
This isn't related to this PR but is the comment still relevant?

Reply:
Yes. This method still only handles boolean flags.